content/blog/meet-our-new-hpe-dev-team-representative-at-hpe-technology-and-solution-summit-2023.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Meet our new HPE DEV Team Representative  at HPE Technology and Solution
```

Comment:
Suggestion: 
Meet the HPE Developer Community team at HPE Technology Summit 2023!